app/src/main/java/com/android/streetworkapp/model/progression/Progression.kt

```diff
@@ -50,3 +50,9 @@ enum class MedalsAchievement(val achievement: Achievement) {
   GOLD(Achievement(R.drawable.place_holder, "Gold medal", emptyList(), "Third medal")),
   PLATINUM(Achievement(R.drawable.place_holder, "Platinum medal", emptyList(), "Last medal"))
```

Comment:
Suggestion: since figma already has the various images, it might be a good idea to put the right ones in instead of the placeholder. I don't know if it's the right PR for that but with figma you just have to right-click on the icon to copy it as a png.

Reply:
I agree but I don't think those on figma really match the bronze, silver, gold, platinum system we have in place. We should probably get some new ones